source/models/tasks_model.py

```diff
@@ -11,12 +12,11 @@ class Task(db.Model):
     date_created = db.Column(DateTime, nullable=False)
```

Comment:
The `task_name` has very few characters for a name, change it to 150 and the description to be at least 300